bokeh/server/tornado.py

```diff
@@ -120,6 +145,7 @@ def __init__(self, applications, prefix, hosts,
             raise ValueError("stats_log_frequency_milliseconds must be > 0")
 
         self._hosts = set(hosts)
+        self._hosts.add('127.*.*.*')
```

Comment:
This is here by default with no way to turn it off right? I don't think it's safe to do that. I don't know offhand what the exploit would be but I don't think it's something we want to allow by default.

Remember, this is not the set of hosts we allow Bokeh to run on, it's the set of hosts we allow a hostile hack to _tell_ Bokeh it is running on. Bokeh then trusts the evil Host and may connect to it from BokehJS, fetch JS from it, fetch data from it, send data to it, who knows.